src/mqt/qecc/cococo/snake_builder.py

```
@@ -0,0 +1,1472 @@
+        unique_handles = [handles[labels.index(label)] for label in unique_legend]
+        plt.legend(unique_handles, unique_legend.keys())
+        plt.show()
+
+    def get_logical_operator_basis(self) -> npt.NDArray[np.int8]:
+        """Generates Logical Operators of the snake.
+
+        Returns:
+            npt.NDArray[np.int8]: logical operators.
+        """
+        hx, hz, _ = self.gen_checks()
+        return CSSCode._compute_logical(np.array(hx), np.array(hz))  # noqa: SLF001
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)